Lombiq.VueJs/Assets/Scripts/helpers/rollup-pipeline.js

```diff
@@ -1,72 +1,113 @@
-const all = require('gulp-all');
+const fs = require('fs');
 const path = require('path');
-const gulp = require('gulp');
-const plumber = require('gulp-plumber');
-const rollup = require('gulp-rollup');
-const rename = require('gulp-rename');
-const uglify = require('gulp-uglify');
-const log = require('fancy-log');
+const { minify } = require('terser');
+const { rollup } = require('rollup');
+
+const { handleErrorObject, handlePromiseRejectionAsError } = require('nodejs-extensions/scripts/handle-error');
+
+function createDirectory(directoryPath) {
+    return fs.existsSync(directoryPath) ? Promise.resolve() : fs.promises.mkdir(directoryPath, { recursive: true });
+}
+
+function handleRollupError(error) {
+    const stack = error?.stack?.split(/[\n\r]+/).map((line) => line.trim()) ?? [];
+    const errorLineRegex = /\.(js|vue):(?<line>\d+)(:(?<column>\d+))?.*/;
+    const lineWithFilePath = stack
+        .filter((line) => line.match(errorLineRegex))
+        .map((line) => line.replace(/^\s*at\s+/, '').trim())[0];
+    const lineMatch = lineWithFilePath?.match(errorLineRegex)?.groups ?? {};
+
+    handleErrorObject({
+        code: 'ROLLUP',
+        path: lineWithFilePath,
+        line: lineMatch ? lineMatch.line : undefined,
+        column: lineMatch ? lineMatch.column : undefined,
```

Comment:
Thanks to line 18, this condition will never be false, so we can simplify this with:
```suggestion
        line: lineMatch.line,
        column: lineMatch.column,
```